package.json

```diff
@@ -35,12 +35,8 @@
   "scripts": {
     "test:unit": "yarn run test:tooling && yarn run test:cc-widgets",
     "test:tooling": "jest --coverage",
-    "test:cc-widgets": "yarn run test:store && yarn run test:widgets",
-    "test:store": "yarn workspace @webex/cc-store test:unit",
-    "test:widgets": "yarn workspace @webex/cc-station-login test:unit && yarn workspace @webex/cc-user-state test:unit",
-    "build": "yarn run build:store && yarn run build:widgets",
-    "build:store": "yarn workspace @webex/cc-store build:src",
-    "build:widgets": "yarn workspace @webex/cc-station-login build:src && yarn workspace @webex/cc-user-state build:src",
+    "test:cc-widgets": "yarn workspaces foreach --all --parallel --exclude webex-widgets --exclude @webex/web-component-samples-app --exclude @webex/react-samples-app run test:unit",
+    "build": "yarn workspaces foreach --all --topological --parallel --exclude webex-widgets --exclude @webex/web-component-samples-app --exclude @webex/react-samples-app  run build:src",
```

Comment:
Excluding webex-widgets, as the tests are passing but it's timing out and failling the build

Reply:
Update: webex-widgets is working fine the failure was because no tests found in cc-widgets, added passWithNoTest flag in jest config

Reply:
UPDATE!! There are no unit tests in widgets